articles/connections/passwordless/regular-web-app-sms/v7/index.md

```diff
@@ -3,7 +3,7 @@ title: Using Passwordless Authentication in a Regular Web App with SMS
 ---
 # Authenticate users with a one-time code via SMS in a Regular Web App
 
-<%= include('../../_older-version-alert' %>
+<%= include('../../../../_includes/_version_warning_auth0js' %>
```

Comment:
script already uses v9 https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-sms/v7#use-your-own-ui (plus there is a Lock section before that)